src/log.cpp

```diff
@@ -10,22 +10,44 @@
 
 Log * Log::mLog;
 
-Log::Log()
-{
-}
 
-Log::~Log()
-{
+void Log::setLogOutput( LogOutput output, const QString& path ) {
+    Log * self = g();
+    switch (output) {
+    case LogOutput::Stderr:
+        self->mOutputFile = stderr;
+        break;
+    case LogOutput::Stdout:
+        self->mOutputFile = stdout;
+        break;
+    case LogOutput::Window:
+        self->mOutputFile = nullptr;
+        break;
+    case LogOutput::File:
+#ifdef    Q_OS_WIN
+        ::fopen_s(&self->mOutputFile, path.toLocal8Bit().constData(), "wt");
+#else  // Q_OS_WIN
+        self->mOutputFile = ::fopen(path.toUtf8().constData(), "w");
```

Comment:
This has to be:

> self->mOutputFile = ::fopen(path.toLocal8bit().constData(), "w");

Otherwise it will not work on Linux when local encoding is not UTF8.

Reply:
Also wondering whether "a" would work better here?